cmd/helm/history.go

```diff
@@ -77,6 +90,7 @@ func newHistoryCmd(c helm.Interface, w io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.Int32Var(&his.max, "max", 256, "maximum number of revision to include in history")
 	f.UintVar(&his.colWidth, "col-width", 60, "specifies the max column width of output")
+	f.StringVarP(&his.outputFormat, "output", "o", "", "prints the output in the specified format ('json' or 'yaml' or defaults to 'table')")
```

Comment:
Perhaps "table" should be a value the switch accepts and should be the default, not ""?